spec/features/hiring_staff_can_save_and_return_later_spec.rb

```diff
@@ -17,7 +17,7 @@
         click_on I18n.t('buttons.create_job')
 
         expect(page.current_path).to eql(job_specification_organisation_job_path)
-        expect(page).to have_content(I18n.t('jobs.create_a_job_title', organisation: school.name))
+        expect(page).to have_content(I18n.t('jobs.create_a_job_title_no_org'))
```

Comment:
Why are we not displaying the organisation's name? Figma:
<img width="503" alt="Screenshot 2020-07-27 at 13 09 24" src="https://user-images.githubusercontent.com/60350599/88540228-76d2d380-d00a-11ea-96d2-ac4944a5bb64.png">
<img width="549" alt="Screenshot 2020-07-27 at 13 09 41" src="https://user-images.githubusercontent.com/60350599/88540230-78040080-d00a-11ea-8f95-dacbd301d165.png">

Reply:
This is because we've changed how we retrieve the name of the organisation for this header. Instead of getting it from the user, we now get it from the vacancy. The vacancy is only created when you reach step 2 of the form. The school or trust name should be displayed on all steps after step 1.